packages/pangea-node-sdk/src/errors.ts

```diff
@@ -65,7 +65,7 @@ export namespace PangeaErrors {
       this.response = response;
     }
 
-    get pangeaResponse(): any {
+    get pangeaResponse(): PangeaResponse<Errors> {
```

Comment:
I'm not sure if this is not an issue with 202 responses where `result` have `accepted_result` and other info